app/templates/account/israel-signup-view.jade

```diff
@@ -0,0 +1,45 @@
+extends /templates/base-flat
+
+block content
+  
+  .container
+    h1.m-t-3.m-b-2.text-center Create Account
```

Comment:
We don't have to translate this until we have confirmed it works, but we will need to do it fairly soon. What do you think the easiest way to have it only translated for Hebrew is? Just get the Hebrew text and use that instead of English, or a comment in the `en.coffee` locale file say not to translate the following section, or some new code to exclude this part from the other locale files when running `copy-i18n-tags.coffee`?